addons/better-terrain/editor/Dock.gd

```diff
@@ -480,20 +553,30 @@ func canvas_input(event: InputEvent) -> bool:
 			initial_click = current_position
 		var type = selected.get_index()
 		
-		if draw_button.button_pressed:
+		if paint_action == PaintAction.LINE:
+			pass
```

Comment:
The way this is structured is correct, but it looks a little odd at a glance, so it could do with a comment explaining it. I figure you're separating out line drawing from the other conditions below?

Reply:
Yeah, for line drawing mode execution happens on release, so this prevents other actions from triggering on press. i could alternatively wrap the other checks in an `if not paint_action == PaintAction.LINE` or return early. for now i've included a comment.

Reply:
Yeah, nothing wrong with it, just thought it'd be good to improve the clarity.